roles/mirror_ocp_release/tasks/main.yml

```diff
@@ -50,6 +50,13 @@
   ansible.builtin.debug:
     var: mor_version
 
+- name: Create temporary working directory
+  ansible.builtin.tempfile:
+    state: directory
+    prefix: mor-
+  register: _mor_tmp
+  notify: "Remove temporary directory"
```

Comment:
We use a handler to remove the temporary directory so we make sure it's run at the end of the play.

The goal is to have it running on successful and failed jobs. For this to work, the play calling this role must activate the flag "force_handlers: true", otherwise the handler will only be run on success.

We chose this approach to a block with an "always" section, so we don't need to add extra tasks to track the failed step properly.